ci/env/install-minimal.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -xe
```

Comment:
What's the reason for this?

Reply:
Kai added this to avoid script didn't fail immediately after the unsuccessful build